### PR TITLE
Feature/implement sass

### PR DESCRIPTION
**Pull request by Alynn022**
PR Template:

Is this a fix or a feature?
- [ ] Bug fix
- [x] Feature
- [ ] Refactor
 
Describe the feature - fix - refactor: implemented sass and added new files for scss

How was this tested?
- [ ] open index.html
- [x] localhost
- [ ] terminal

Where should the reviewer start - test: css

Additional comments:

### CODE REVIEW

File: src/css/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+$primary-color: #3E673C;
+$border-color: #5D9A5B;
+$text-bg-color: #e8f1f2;
+$button-font: 'Roboto', sans-serif;
+
+
+$gradient: linear-gradient(to right, #1B3627 0%, #3E673C 100%);
+
+@mixin button-options() {
```
Comment (StevieAmb) [inline src/css/_variables.scss:9]: This is so cool, I definitely want to look at this more.

File: src/css/index.scss [hunk 2]
```diff
@@ -87,19 +97,53 @@ input {
...
+.add-recipe-to-cook-button,
+.remove-recipe-to-cook-button,
+.unfavorite-button,
+.favorite-button {
+  align-self: flex-end;
+  height: 12vh;
+  width: 10vw;
+}
+
+.dropbtn:hover,
```
Comment (StevieAmb) [inline src/css/index.scss:123]: I love this stacking that can happen and the focus is awesome!

File: src/scripts.js [hunk 3]
```diff
@@ -235,7 +235,7 @@ const showAllRecipes = () => {
     allRecipesView.insertAdjacentHTML('afterbegin', `
     <article class="recipes-views" id= ${recipe.id}>
     <h1>${recipe.name}</h1>
-    <img src="${recipe.image}">
+    <img src="${recipe.image}" tabindex= "0">
```
Comment (StevieAmb) [inline src/scripts.js:238]: This is a super cool way to add tabbing to an element being pushed into the DOM.

**Review by StevieAmb — APPROVED** [review]

**Review by lschnell8 — APPROVED** [review]
This looks awesome! Great job with the SCSS implementation. I do wonder if you tried out a @normalize in lieu of the @reset? If not, we should totally explore the differences!

**Pull request merged**